src/python/WMCore/Database/DBExceptionHandler.py

```diff
@@ -41,5 +41,7 @@ def wrapper(*args, **kwargs):
 
             if raiseFlag:
                 raise
+            # returns tuples of 3 since timeFunc returns (time to spend, result, func.__name__)
+            return None, None, None
```

Comment:
How about replacing the `break` above by this line? Then we probably don't need the raiseFlag either.
Last but not least, I'd return `0` for time spent, just to avoid issues with data type and etc that is inserted on the db.